plugins/tekton/src/components/PipelineRunList/PipelineRunListSearchBar.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import useDebounce from 'react-use/lib/useDebounce';
+
+import {
+  FormControl,
+  IconButton,
+  Input,
+  InputAdornment,
+  makeStyles,
+} from '@material-ui/core';
+import Clear from '@material-ui/icons/Clear';
+import Search from '@material-ui/icons/Search';
+
+type PipelineRunListSearchBarProps = {
+  pipelineRuns?: any[];
```

Comment:
nit: can we have type for it?

```suggestion
  pipelineRuns?: PipelineRunKind[];
```